subsys/bluetooth/controller/ll_sw/ctrl.c

```diff
@@ -2299,8 +2299,11 @@ static inline u8_t isr_rx_conn_pkt_ctrl_dle(struct pdu_data *pdu_data_rx,
 
 			/* No change in effective octets or time */
 			if (eff_tx_octets == _radio.conn_curr->max_tx_octets &&
+#if defined(CONFIG_BT_CTLR_PHY)
 			    eff_tx_time == _radio.conn_curr->max_tx_time &&
-			    eff_rx_time == _radio.conn_curr->max_rx_time) {
+			    eff_rx_time == _radio.conn_curr->max_rx_time &&
+#endif /* CONFIG_BT_CTLR_PHY */
```

Comment:
Would using IS_ENABLED() inside the if-statement make this kind of constructions cleaner/more readable? Not just related to this place - I've seem similar things in quite many places.

Reply:
I've at least strived toward using IS_ENABLED() whenever possible. The situations when it can't be used is when some variable or struct member doesn't exist if the option isn't enabled, however that doesn't seem to be the case here.

Reply:
@jhedberg I have the max_tx_time and max_rx_time conditionally compiled out in the struct connection, I am not able to use IS_ENABLED in these cases.